json/store.go

```diff
@@ -0,0 +1,126 @@
+package json
+
+import (
+	"encoding/json"
+	"fmt"
+	"go.mozilla.org/sops"
+	"go.mozilla.org/sops/kms"
+	"go.mozilla.org/sops/pgp"
+	"time"
+)
+
+type JSONStore struct {
+}
+
+func (store JSONStore) Load(in string) (sops.TreeBranch, error) {
+	var branch sops.TreeBranch
+	err := json.Unmarshal([]byte(in), branch)
+	if err != nil {
+		return branch, fmt.Errorf("Could not unmarshal input data: %s", err)
+	}
+	for i, item := range branch {
+		if item.Key == "sops" {
+			branch = append(branch[:i], branch[i+1:]...)
+		}
+	}
+	return branch, nil
+}
+
+func (store JSONStore) Dump(tree sops.TreeBranch) (string, error) {
+	out, err := json.Marshal(tree)
+	if err != nil {
+		return "", fmt.Errorf("Error marshaling to json: %s", err)
+	}
+	return string(out), nil
```

Comment:
I think we should return `[]byte` and not `string` here, because that's what all marshalers do and it would be useful for other clients that want an iowriter.